README.md

```diff
@@ -267,7 +268,7 @@ Make sure to see [CONTRIBUTING.md](/CONTRIBUTING.md) for instructions on contrib
 * [Fortran77](f/fortran77.f77)
 * [Fourier](f/fourier.fourier)
 * [Fractran++](f/fractran%2B%2B.fpp)
-* [FreeBrain](f/freebrain.frbr)
+* [Freebrain](f/freebrain.frbr)
```

Comment:
Don't make unnecessary changes.

Reply:
I just ran update_list.py, I did not change anything in readme.md manually!